docs/source/tools/visual.rst

```diff
@@ -0,0 +1,32 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+DAML Contract Visulization
+#########
+
+Introduction
+************
+
+You can generate visual graphs for DAML contracts visual command via command line tool. This program generates dot file which in turn can fed in to graphviz engine to produce an image.
+
+Trying it out
+*************
```

Comment:
Do we need an empty line below this heading for the rendering to work? Same below.

Reply:
Nope, not needed it. Removed